Formula/scrcpy.rb

```diff
@@ -0,0 +1,78 @@
+class Scrcpy < Formula
+  desc "Display and control your Android device"
+  homepage "https://github.com/Genymobile/scrcpy"
+  url "https://github.com/Genymobile/scrcpy/archive/v1.0.tar.gz"
+  sha256 "fda84b2fc6a60bf808f7b87f7c9a985a446bad69d4de8e63e9ce699871f8026f"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "ffmpeg"
+  depends_on "sdl2"
+
+  resource "prebuilt-server" do
+    url "https://github.com/Genymobile/scrcpy/releases/download/v1.0/scrcpy-server-v1.0.jar"
+    sha256 "b573b06a6072476b85b6308e3ad189f2665ad5be4f8ca3a6b9ec81d64df20558"
+  end
+
+  def install
+    r = resource("prebuilt-server")
+    r.verify_download_integrity(r.fetch)
+    cp r.fetch, buildpath/"prebuilt-server.jar"
+
+    meson_args = %w[x --buildtype release --strip -Db_lto=true]
```

Comment:
What does this lone `x` mean?
If it's an arbitrary build directory, can be use `build` instead please?